staging/src/k8s.io/apimachinery/pkg/conversion/unstructured/testing/converter_test.go

```diff
@@ -127,7 +127,7 @@ func doRoundTrip(t *testing.T, item interface{}) {
 		return
 	}
 	unmarshalledObj := reflect.New(reflect.TypeOf(item).Elem()).Interface()
-	err = json.Unmarshal(data, &unmarshalledObj)
+	err = json.Unmarshal(data, unmarshalledObj)
```

Comment:
Forgot to mention - this is a bug in the test. `reflect.New` returns a wrapped pointer to a freshly allocated type. Passing a pointer to the interface that contains that pointer is... pointless :)